Code/Tools/ProjectManager/Source/GemCatalog/GemSortFilterProxyModel.h

```diff
@@ -69,7 +69,7 @@ namespace O3DE::ProjectManager
         const QSet<QString>& GetFeatures() const { return m_featureFilter; }
         void SetFeatures(const QSet<QString>& features) { m_featureFilter = features; InvalidateFilter(); }
 
-        bool GetUpdateAvailable() const { return m_updateAvailableFilter; }
+        const bool GetCompatibleFilterFlag() const { return m_compatibleOnlyFilter; }
```

Comment:
A const value type shouldn't be returned.
The compiler ignores it anyway and returns just a `bool`
```suggestion
        bool GetCompatibleFilterFlag() const { return m_compatibleOnlyFilter; }
```